src/index.html

```diff
@@ -11,8 +12,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__pic"
+          src="images/imac.jpeg"
+          alt="Imac picture"
+        />
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__reviews card__reviews--count">Reviews: 5</span>
```

Comment:
```suggestion
          <span class="card__reviews-count">Reviews: 5</span>
```
make it an element instead of modificator